# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentChild.h
# Commit: df07abc82381
# Full Hash: df07abc82381d2a3067b7de63867a0181f497519
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2025-03-26 09:13:23
# Regressor Bug: 1899503
# File Overlap Count: 2
# Description:
#   Bug 1899503 - P2 PRemoteWorkerDebuggerMangaer binding implementation. r=asuth
#   
#   This patch implements the PRemoteWorkerDebuggerManager binding.
#   
#   PRemoteWorkerDebuggerManager is the IPC that handles the registration/unregistration of remote debuggers.
# ==============================================================================

diff -r 5d39af040a61 -r df07abc82381 dom/ipc/ContentChild.h
--- a/dom/ipc/ContentChild.h	Tue Mar 25 14:33:27 2025 +0000
+++ b/dom/ipc/ContentChild.h	Tue Mar 25 14:33:27 2025 +0000
@@ -372,7 +372,8 @@
   const nsACString& GetRemoteType() const override;
 
   mozilla::ipc::IPCResult RecvInitRemoteWorkerService(
-      Endpoint<PRemoteWorkerServiceChild>&& aEndpoint);
+      Endpoint<PRemoteWorkerServiceChild>&& aEndpoint,
+      Endpoint<PRemoteWorkerDebuggerManagerChild>&& aDebuggerChildEp);
 
   mozilla::ipc::IPCResult RecvInitBlobURLs(
       nsTArray<BlobURLRegistrationData>&& aRegistations);